Azure-Samples/Cognitive-Services-Voice-Assistant
# Refactor UI

## Purpose
* Refactor existing UI to reflect Design Team's iteration
* UI consists of three Panes (Controls, Logs, and Chat)
* Merged Travis' MP3 support changes
* Logs are now colored based on Log Level

## WIP
* Sending Text Input
* Updating Chat window

## Does this introduce a breaking change?
[x] Yes
[ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

## How to Test
* Build the solution
* Click Filter Views and toggle each of the three selections - window should resize to display selected Panes.

![UWPVoiceAssistantCurrentUI_April16](https://user-images.githubusercontent.com/38671931/79496659-4aef8d00-7fdb-11ea-8199-f746357e3e2e.png)
